### PR TITLE
[ADD] New job entry (Chess.com)

### PR DESCRIPTION
**Pull request by KenyStev**
This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [X] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [] You are an employee of the company mentioned and confirm all included details are correct.
- [X] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [X] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [X] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [X] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [X] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [X] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [X] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [X] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: company-profiles/chess.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Chess
+
+## Company blurb
+
+Chess.com is a global team of passionate chess fans and developers. You can meet us here and read about our company here. We're always looking for great people to join the team. We have a healthy work environment focused on well-being, respect, and memes/emotes. We communicate with Slack and positive energy beams.
```
Comment (KieranRobson) [inline company-profiles/chess.md:5]: You say here, but havent hyperlinked to anything was that a typo or is the blurb the `here`

Comment (KenyStev) [reply company-profiles/chess.md]: yeah I missed the hyperlinks, thanks!

**Review by KieranRobson — APPROVED** [review]
Looks great!

**Comment by dougaitken** [comment]
Hey there, thanks for contributing, it is appreciated

**Pull request merged**